chandra_models/xija/minusyz/minusyz_spec.json

```diff
@@ -1,1262 +1,1269 @@
+                    0.1,
+                    0.1,
+                    0.1,
+                    0.1,
                     0.1
                 ]
-            ], 
+            ],
             "init_kwargs": {
-                "ampl": 0.003851, 
-                "epoch": "2013:182:12:00:00", 
-                "tau": 365, 
+                "ampl": 0.003851,
+                "epoch": "2013:182:12:00:00",
+                "tau": 365,
                 "var_func": "linear"
-            }, 
+            },
             "name": "solarheat__tephin_0"
-        }, 
+        },
         {
-            "class_name": "Coupling", 
+            "class_name": "Coupling",
             "init_args": [
-                "tephin", 
+                "tephin",
                 "tephin_0"
-            ], 
+            ],
             "init_kwargs": {
                 "tau": 100.0
-            }, 
+            },
             "name": "coupling__tephin__tephin_0"
-        }, 
+        },
         {
-            "class_name": "Coupling", 
+            "class_name": "Coupling",
             "init_args": [
-                "tephin", 
+                "tephin",
                 "pmtank3t"
-            ], 
+            ],
             "init_kwargs": {
                 "tau": 100.0
-            }, 
+            },
             "name": "coupling__tephin__pmtank3t"
-        }, 
+        },
         {
-            "class_name": "Coupling", 
+            "class_name": "Coupling",
             "init_args": [
-                "tephin", 
+                "tephin",
                 "tcylaft6"
-            ], 
+            ],
             "init_kwargs": {
                 "tau": 100.0
-            }, 
+            },
             "name": "coupling__tephin__tcylaft6"
         }
-    ], 
-    "datestart": "2012:183:12:00:32.816", 
-    "datestop": "2014:182:11:51:20.816", 
-    "dt": 328.0, 
+    ],
+    "datestart": "2012:183:12:00:32.816",
+    "datestop": "2014:182:11:51:20.816",
+    "dt": 328.0,
     "gui_config": {
-        "filename": "/home/mdahmer/ThermalModels/xija/chandra_models/chandra_models/xija/minusyz/minusyz_spec_linear2.json", 
+        "filename": "/home/mdahmer/ThermalModels/xija/chandra_models/chandra_models/xija/minusyz/minusyz_spec_linear2.json",
         "plot_names": [
-            "tephin data__time", 
+            "tephin data__time",
             "tephin resid__time"
-        ], 
+        ],
         "set_data_vals": {
-            "pmtank3t_0": 30.0, 
-            "tcylaft6_0": 30.0, 
+            "pmtank3t_0": 30.0,
+            "tcylaft6_0": 30.0,
             "tephin_0": 30.0
-        }, 
+        },
         "size": [
-            1890, 
+            1890,
             1004
         ]
-    }, 
-    "mval_names": [], 
-    "name": "minusyz", 
+    },
+    "limits": {
+        "tcylaft6": {
+            "planning.warning.high": 130,
+            "planning.warning.high.matlab_identifier": "yellow_limit",
+            "unit": "degF"
+        }
+    },
```

Comment:
the MATLAB tools use the file `tcylaft6_spec.json` for this model.  It looks like they don't use this file at all.  Should the limits for tcylaft6 be added to it's spec file directly?

Also same question as in other places about naming convention.  Should this be a `caution` instead of a `warning`?